### PR TITLE
An active logged out user wants to sign up for an offer

### PR DESCRIPTION
**Pull request by gruszks**
**Story / Bug id:** https://github.com/stxnext-csr/volontulo/issues/495
**Reference person:** @gruszks
**Description:**

An active logged out user wants to sign up for an offer

### CODE REVIEW

File: apps/volontulo/tests/views/test_offers.py [hunk 1]
```diff
@@ -666,16 +666,16 @@ def test_offers_join_valid_form_and_anonymous_user(self):
         response = self.client.post('/offers/volontulo-offer/{}/join'.format(
             self.offer.id
         ), {
-            'email': u'anon@example.com',
-            'phone_no': u'+42 42 42 42',
-            'fullname': u'Mister Anonymous',
-            'comments': u'Some important staff.',
+            'email': 'anon@example.com',
```
Comment (filipgorczynski) [inline apps/volontulo/tests/views/test_offers.py]: Code duplication so maybe good idea would be to move this dictionaries to one place?

Comment (gruszks) [inline apps/volontulo/tests/views/test_offers.py]: Done.

**Comment by filipgorczynski** [comment]
Looks :white_check_mark:

**Comment by radoslawroszkowiak** [comment]
+1

File: apps/volontulo/views/offers.py [hunk 2]
```diff
@@ -411,17 +411,10 @@ def post(request, slug, id_):  # pylint: disable=unused-argument
...
-                    return render(
-                        request,
-                        'offers/offer_apply.html',
-                        {
-                            'offer': offer,
-                            'form': form,
-                            'volunteer_user': UserProfile(),
-                        }
+                    return redirect(
+                        reverse('login') + '?next={}'.format(request.path)
```
Comment (tomaszwozniak) [inline apps/volontulo/views/offers.py:417]: please use format for reverse as well

**Comment by tomaszwozniak** [comment]
oh lol nvm

**Pull request merged**